spdx-utils/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -289,27 +293,25 @@ class SpdxExpressionTest : WordSpec() {
 
         "disjunctiveNormalForm()" should {
             "not change an expression already in DNF" {
-                val spdxExpression = "(a AND b) OR (c AND d)".toSpdx()
-
-                spdxExpression.disjunctiveNormalForm() shouldBe spdxExpression
+                "(a AND b) OR (c AND d)".toSpdx().disjunctiveNormalForm() shouldBeString "(a AND b) OR (c AND d)"
```

Comment:
Shouldn't the expected / normalized string also drop the parentheses here, like in the other examples, as `AND` binds stronger than `OR` by default?